closed/adds/jdk/src/share/native/jdk/crypto/jniprovider/NativeCrypto.c

```diff
@@ -44,17 +141,192 @@ typedef struct OpenSSLMDContext {
 static void printErrors(void) {
     unsigned long errCode = 0;
 
-    fprintf(stderr, "An error occurred\n");
-    while(0 != (errCode = ERR_get_error()))
+    fprintf(stderr, "An OpenSSL error occurred\n");
+    while(0 != (errCode = (*OSSL_get_error)()))
     {
         char err_str[120];
-        ERR_error_string_n(errCode, err_str, (sizeof(err_str) / sizeof(char)));
-        fprintf(stderr, "Generating error message\n" );
+        (*OSSL_error_string_n)(errCode, err_str, (sizeof(err_str) / sizeof(char)));
         fprintf(stderr, "%s\n", err_str);
     }
     fflush(stderr);
 }
 
+/*
+ * Class:     jdk_crypto_jniprovider_NativeCrypto
+ * Method:    loadCrypto
+ * Signature: ()V
```

Comment:
Please correct the signature: should be `()I`.

Reply:
Done